serviceregistry-server/src/main/java/no/difi/meldingsutveksling/serviceregistry/service/elma/ElmaStagingConfig.java

```diff
@@ -29,7 +29,7 @@ public TransportProfile getTransportProfile() {
     public LookupClient getElmaLookupClient() throws PeppolLoadingException {
         Mode mode = Mode.of(Mode.TEST);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "mode" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1481)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this useless assignment to local variable "mode". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1854)